### PR TITLE
Updating service versions for blobs and datalake

### PR DESCRIPTION
**Pull request by ibrahimrabab**
# Description

Updating service versions for blobs and datalake to use 08/06/2022

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### CODE REVIEW

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java [hunk 1]
```diff
@@ -94,6 +94,6 @@ public String getVersion() {
      * @return the latest {@link BlobServiceVersion}
      */
     public static BlobServiceVersion getLatest() {
-        return V2021_06_08;
+        return V2021_08_06;
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java:97]: I think we also need to update this in Share File and any other storage package that has a `*ServiceVersion.java` file.

Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java:97]: don't forget about https://github.com/Azure/azure-sdk-for-java/blob/5087bf2173419f89f7e381d267f6d2b8b7de750f/sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java#L90-L91

Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java:97]: and https://github.com/Azure/azure-sdk-for-java/blob/5087bf2173419f89f7e381d267f6d2b8b7de750f/sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java#L219

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**